src/__tests__/RadioButton.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { mount } from 'enzyme'
```

Comment:
I think you should be able to swap out all of your `mount`s with `shallow`s, since you're only testing the root wrapper.